### DISCUSSION TITLE
Why BeautifulDate and not BeautifulDateTime?

### DISCUSSION THREAD

**Discussion by N-Demir** [kuzmoyev/beautiful-date#26] [General]
Hi! Yeah curious about why BeautifulDate is just a date and not time as well?

**Comment by kuzmoyev** [comment]
The `BeautifulDate` has this additional functionality (compared to a normal date) that allows it to be converted to a datetime with indexing, e.x. `d[22:44]`.

But I haven't came up with any interesting functionality to add to the datetime, so haven't introduced the dedicated class for that. Do you have any ideas for a `BeautifulDateTime`?
